buildSrc/src/main/groovy/com/uber/okbuck/rule/android/AndroidTestRule.groovy

```diff
@@ -40,6 +40,7 @@ final class AndroidTestRule extends AndroidRule {
                 aptDeps,
                 providedDeps,
                 aidlRuleNames,
+                null,
```

Comment:
Can you not cherry pick and test it locally? Your PR has not merged so far either :)